stake-pool/cli/src/main.rs

```diff
@@ -1257,15 +1257,21 @@ fn prepare_withdraw_accounts(
 
             (
                 transient_stake_account_address,
-                validator.transient_stake_lamports,
```

Comment:
We'll need to do `validator.transient_stake_lamports.saturating_sub(min_balance)`, otherwise 0 will give *a lot* of SOL :laughing: